docs/content/getting-started/install.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: "Installation"
-description: "Download and install the hof cli tool"
-brief: "Download and install the hof cli tool"
+description: "Download and install the hof CLI tool."
+brief: "Download and install the hof CLI tool."
 weight: 3
 ---
 
 {{<lead>}}
-__hof__ is available for all major operation systems and architectures.
+__hof__ is available for all major operating systems and architectures.
```

Comment:
I wonder if there is a grammar checker that we could run to catch more of my wording errors :] 🤔

Reply:
true, this wasn't flagged when i scanned for typos because isn't a typo, a grammar scan is a good idea, will check about a CI for that

Reply:
saw this one, will try it out in a test repo to see if works
https://github.com/awreck/proofread-action